app/services/rapid_ready_for_decision/hypertension_special_issue_manager.rb

```diff
@@ -23,13 +23,16 @@ def add_special_issue
     RRD_CODE = 'RRD'
 
     def hypertension_increase?(disability)
-      RapidReadyForDecision::ProcessorSelector.disability_increase?(disability, HYPERTENSION_CODE)
+      # Does the same as RapidReadyForDecision::ProcessorSelector.disability_increase?(disability, HYPERTENSION_CODE)
+      # except using symbols as the keys instead of strings
+      disability[:diagnosticCode] == HYPERTENSION_CODE &&
+        disability[:disabilityActionType].downcase == 'increase'
```

Comment:
Using `downcase` to revert back to the original code even though claims in prod use uppercase "INCREASE".